namespace_conn.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/googollee/go-socket.io/parser"
 )
 
-// Namespace
+// Namespace describes a socket.io room.
```

Comment:
`Namespace describes a communication channel that allows you to split the logic of your application over a single shared connection.`

A namespace is not a room, see https://socket.io/docs/v2/namespaces/.